src/Navbar.jsx

```diff
@@ -38,7 +51,18 @@ const Navbar = () => {
           <StyledLink to="/users/register">Register</StyledLink>
         </StyledListItem>
         <StyledListItem>
-          <StyledLink to="/users/login">Login</StyledLink>
+          {token ? (
+            <Button
+              navLink
+              onClick={() => {
```

Comment:
Would `onClick={ logout }` work?

Reply:
technically they are different signatures as there is an event passed with onClick, they are just ignoring it here. So that would work, but typescript might start complaining etc.
Generally I would say its preferable to buffer calls in this way unless there is a good reason.